CONTRIBUTING.md

```diff
@@ -13,4 +13,5 @@ please adhere to the following rules:
 * (re-)write tests to verify the behaviour of your additions
 * look through the open issues and refer to them if they relate to your PR
 * look through the open PRs and refer to them if they relate to your PR
+* we enforce a coding style, please validate your contribution before pushing with the correspondig makefile targets (see `make help`)
```

Comment:
maybe be more on point, like `make format`, or what ever the current command will be

Reply:
should I rename it to `make format` & `make check-format` ?

Reply:
my intention was, that the correct command for formatting is mentioned here. I just did not look up, what the correct command would be

Reply:
so you are fine with `make uncrustify` & `make check-uncrustify` ?

Reply:
i would prefer something along the lines of `make format` and `make check-format`, but if you prefer the uncrustify commands, i am fine with it